core/server/data/schema/schema.js

```diff
@@ -308,5 +308,12 @@ module.exports = {
         session_data: {type: 'string', maxlength: 2000, nullable: false},
         created_at: {type: 'dateTime', nullable: false},
         updated_at: {type: 'dateTime', nullable: true}
+    },
+    mobiledoc_revisions: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        post_id: {type: 'string', maxlength: 24, nullable: false, index: true},
```

Comment:
> index: true

Did you double check that? Looks like the wrapper does not support index currently

Reply:
confirming it's not working, the schema in mysql looks like this after migration:
```
| mobiledoc_revisions | CREATE TABLE `mobiledoc_revisions` (
  `id` varchar(24) NOT NULL,
  `post_id` varchar(24) NOT NULL,
  `mobiledoc` longtext,
  `created_at_ts` bigint(20) NOT NULL,
  `created_at` datetime NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4
```

Reply:
@rishabhgrg You just removed it? Why is that?

Reply:
@kirrg001 We just tested it with @gargol, it does not supports index right now so removed it quickly. We were just discussing if we should add index support as part of these changes or we leave it for now

Reply:
@kirrg001 based on https://github.com/TryGhost/Ghost/blob/master/core/server/data/schema/commands.js#L8-L46, we don't seem to support indexes at the moment right?

Reply:
Yeah based on my assumption and based on the confirmation [comment](https://github.com/TryGhost/Ghost/pull/9934#discussion_r221980335) from @gargol, i noticed it's not supported.

@rishabhgrg 
Just removing "index: true", force push the commit and not leaving a comment why you decided that way, is a little invisible. The discussion was even hided after force push, which makes is hart to keep track of what's going on.

I'd suggest to double check knex docs and see what's missing to support the index?

Reply:
@kirrg001 Hmm yeah sorry was bit too over-reactive and pushing changes too fast to push this for today's release keeping a single commit, its not helpful 😅Will look into knex docs for index feature, should I meanwhile revert anything for cleaner history or leave as is for now?

Reply:
>  should I meanwhile revert anything for cleaner history or leave as is for now?

no history looks fine 👍

Reply:
@rishabhgrg do you need help?

Reply:
Just finished updating, `index` is in place in wrapper using knex command, tested on local mysql and it seems to work
<img width="900" alt="screen shot 2018-10-02 at 9 48 28 pm" src="https://user-images.githubusercontent.com/3461456/46362070-ef75dc80-c68c-11e8-99af-ee9cca3c823c.png">

Reply:
Cool. Where is this handled in our command wrapper? Can you pls show me the code section? Thanks 👍

Reply:
💃